client/src/app/actions/api.js

```diff
@@ -0,0 +1,27 @@
+import axios from 'axios';
+import apiRoute from './apiRoute'
+
+// export default {
+// user:{
+//     login: credentials =>
+//     axios.post(`${apiRoute}/users/signin`,
+//     {credentials}).then(res => res.data.user)
+// }
+
+// };
+
+
+export default {
+    user:{
+        login: credentials =>
+        axios.post('/api/v1/users/signin',
+        credentials)
+    }
+}
+    //     .then((res,err) => 
```

Comment:
normally a `.then` will take only the `response`, the `err` will be caught in the `.catch`

Reply:
sorry i commented that out i was testing axios out. I am reading up on promises